pipelines/tekton/aiedge-e2e/tasks/copy-model-from-pvc.yaml

```diff
@@ -11,14 +11,14 @@ spec:
   - name: copy-model-from-pvc
     image: quay.io/opendatahub/kserve-storage-initializer:v0.11
     script: |
-      SOURCE_PATH="$(workspaces.source-workspace.path)/model_dir/$(params.model-name)"
+      SOURCE_PATH="$(workspaces.source-workspace.path)/model_dir-$(params.model-name)/$(params.model-name)"
```

Comment:
I'm sorry for the confusion. Please also make change in the Makefile when uploading the model to PVC, otherwise this will not work.